src/lib/wordpress-provider/types.ts

```diff
@@ -42,6 +42,8 @@ export interface WordPressServerProcess {
 }
 
 export interface WordPressProvider {
+	readonly PROVIDER_TYPE: string;
```

Comment:
Nit: we can consider renaming this property to NAME or PROVIDER_NAME as we don't expect to have more providers of the same type.

```suggestion
	readonly NAME: string;
```

It also aligns better with other constants of the code like `readonly PROVIDER_TYPE = PLAYGROUND_CLI_PROVIDER_NAME;`